lib/Model/CommonTypeMethods.cpp

```diff
@@ -351,6 +467,37 @@ const model::Type &Common<CRTP>::stripPointer() const {
   return *asPointer().PointeeType();
 }
 
+template<typename CRTP>
+void Common<CRTP>::stripConstness() {
```

Comment:
I'm a bit confused about this method. What's its use case?

It seems to strip `const`ness kinda recursively but not really (what about `typedefs` of `typedefs` of `typedefs`?).

Reply:
What do you mean? Tt is recursive! It calls `stripConstness` of the typedef's `UnderlyingType`.